docs/known_issues.md

```diff
@@ -38,3 +38,19 @@ Calls to form helpers such as `form_with` in ViewComponents [don't use the defau
   <%= f.text_field :name %>
 <% end %>
 ```
+
+## Unwanted whitespace
```

Comment:
🤔 I wonder if this would be better off on the Templates docs page, as this is a feature!